src/index.js

```diff
@@ -1,8 +1,16 @@
 // Core
-export createRedux from './createRedux';
-export createDispatcher from './createDispatcher';
+import createRedux from './createRedux';
+import createDispatcher from './createDispatcher';
 
 // Utilities
-export composeMiddleware from './utils/composeMiddleware';
-export composeStores from './utils/composeStores';
-export bindActionCreators from './utils/bindActionCreators';
+import composeMiddleware from './utils/composeMiddleware';
+import composeStores from './utils/composeStores';
+import bindActionCreators from './utils/bindActionCreators';
+
+export {
+	createRedux,
+	createDispatcher,
+	composeMiddleware,
+	composeStores,
+	bindActionCreators
+};
```

Comment:
ESLint will not complain if you use `"parser": "babel-eslint"` like in the original config.
It is essential!

_But_ I heard that `export` syntax is not going to make it in ES7 after all. So I'm fine with replacing this with `export { default as createDispatcher } from './createDispatcher`, etc.